docs/content/configuration/basics.md

```diff
@@ -0,0 +1,650 @@
+
+* When backing up:
+  * MariaDB: `--no-tablespaces`
+  * MySQL: `--no-tablespaces`
+  * PostgreSQL: `-Z 5 --lock-wait-timeout=120`
+  * SQLite: `-bail`
+* When restoring:
+  * MariaDB: None
+  * MySQL: None
+  * PostgreSQL: `-j 2 --clean --if-exists --disable-triggers`
+  * SQLite: None
+
+## Anonymizer paths
+
+<div class="symfony">
+
+By default, the *DbToolsBundle* will look for custom *anonymizers* in 2 directories:
+
+* `%kernel.project_dir%/vendor/makinacorpus/db-tools-bundle/src/Anonymizer`
+* `%kernel.project_dir%/src/Anonymizer`
+
+If you want to put custom anonymizers in another directory or if you want to load
+a pack of anonymizers from an external library, you can modify/add paths:
+
+
+```yml
+# config/packages/db_tools.yaml
+db_tools:
+    anonymizer_paths:
+        - '%kernel.project_dir%/src/Anonymizer'
+        - '%kernel.project_dir%/vendor/myAnonymizerProvider/anonymizers/src'
+        # ...
+```
+</div>
+<div class="standalone">
+
+By default, the *DbToolsBundle* when used as a standalone CLI tool will not lookup for
+custom *anonymizers*.
+
+If you want to write custom anonymizers, in order for them to be found, you must specify
+paths where the source code lies:
+
+
+```yml
+# db_tools.yaml
+db_tools:
+    anonymizer_paths:
+        - './src/Anonymizer'
+        - './vendor/myAnonymizerProvider/anonymizers/src'
+        # ...
+```
+</div>
+
+:::tip
+Core provided anonymizers and anonymizers that are in packs installed using composer
+will always be looked-up.
+:::
+
+:::warning
+Packs must be installed using composer: *DbToolsBundle* uses composer generated metadata
+about installed packages to find them.
+:::
+
+## Anonymization
+
+<div class="symfony">
+
+Per default, the **DbToolsBundle** will only look for anonymization configurations from PHP attributes on Doctrine Entities.
+
+But *DbToolsBundle* does not necessary need Doctrine ORM to anonymize your data, it can do it just with a DBAL connection.
+In this case (or if you prefer YAML over attributes): you can configure the *DbToolsBundle* to look for anonymization
+configurations in a YAML file:
+
+```yml
+# config/packages/db_tools.yaml
+db_tools:
+
+    # When you have a single connection, and a single file:
+    anonymization_files: '%kernel.project_dir%/config/anonymizations.yaml'
+
+    # Or with multiple connections:
+    anonymization_files:
+        connection_one: '%kernel.project_dir%/config/anonymizations/connection_one.yaml'
+        connection_two: '%kernel.project_dir%/config/anonymizations/connection_two.yaml'
+
+    # Each connection may have multiple files:
+    anonymization_files:
+        connection_one:
+            - '%kernel.project_dir%/config/anonymizations/connection_one_1.yaml'
+            - '%kernel.project_dir%/config/anonymizations/connection_one_2.yaml'
+        # ...
+```
+
+</div>
+<div class="standalone">
+
+You need to register your anonymization configuration for the anonymization feature to work:
```

Comment:
May be you can start with the simple version with anonymization directly in the config file, and then tell that one can choose to extract this conf to another file

Reply:
I was thinking that this part was not right. Your suggestion seems good.